src/models/__tests__/message.test.ts

```diff
@@ -790,11 +792,36 @@ describe("getContentFromBlob", () => {
     getBlobAsTextSpy.mockReset();
   });
 
-  it("should fail with an error when the blob cannot be retrieved", async () => {
-    const err = Error();
+  it("should get a none if blob do not exists", async () => {
+    const err = {
+      name: "StorageError",
+      message:
+        "The specified blob does not exist.\nRequestId:27149c2d-001e-000e-2d04-a12766000000\nTime:2022-07-26T15:27:59.0934919Z",
+      code: "BlobNotFound",
+      statusCode: 404,
+      requestId: "27149c2d-001e-000e-2d04-a12766000000"
+    };
     const getBlobAsTextSpy = jest
-      .spyOn(azureStorageUtils, "getBlobAsText")
-      .mockReturnValueOnce(Promise.resolve(E.left(err)));
+      .spyOn(azureStorageUtils, "getBlobAsTextWithError")
+      .mockReturnValueOnce(() => TE.left(err));
```

Comment:
We could add another test with error having a different error code

Reply:
done. thank you!